public/app/features/variables/state/actions.ts

```diff
@@ -78,14 +78,21 @@ import isEqual from 'lodash/isEqual';
 
 export const initDashboardTemplating = (list: VariableModel[]): ThunkResult<void> => {
   return (dispatch, getState) => {
-    let orderIndex = 0;
     for (let index = 0; index < list.length; index++) {
       const model = list[index];
       if (!variableAdapters.getIfExists(model.type)) {
         continue;
       }
 
-      dispatch(addVariable(toVariablePayload(model, { global: false, index: orderIndex++, model })));
+      const global = false;
+      // const index = getNewVariabelIndex(getState());
+      const identifier: VariableIdentifier = { type: 'adhoc', id: model.id };
+
+      dispatch(
+        addVariable(
+          toVariablePayload<AddVariable>(identifier, { global, model, index })
+        )
+      );
```

Comment:
This change does not make any sense to mee, whats the point of it? what's the point of global variable?  almost every line changed here looks very strange

Reply:
Yes, agreed this will not make sense to the bug.